README.md

```diff
@@ -8,9 +8,11 @@ Here's the [YouTube Video](https://youtu.be/9AXP7tCI9PI).
 
 Install [Langchain](https://github.com/hwchase17/langchain) and other required packages.
 ```
-pip install langchain openai chromadb tiktoken unstructured
+pip install langchain openai chromadb tiktoken unstructured pypdf pythondotenv wrapt==1.15
```

Comment:
remove extra depedencies like wrapt, pypdf?  doesn't seem necessary for .env change...

Reply:
I'm not sure about `pypdf`, I probably needed to install it, but now I cannot find out, because I installed most of the missing packages into my site-packages, so my environment is not vanilla anymore. 
As for `wrapt`, it might be a dependency of a higher level package...

My current research aims the missing package `detectron2`, because I get prompted upon bootup and the PDF conversion seems to be pretty dumb at the moment. Certainly, the sample PDF is read easily, but I tried another one and not even the questions about the headlines are answered.

[Detectron2 installation guide](https://detectron2.readthedocs.io/en/latest/tutorials/install.html)